web/lib/plugins/seneca-dojo-preloader.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Prepare an object usable for openGraph containing information for a dojo
+ * @param  {Object}   request
+ * @param  {Function} cb      Callback assigning the preloaded object and continuing
+ */
+var translater = require('../fn/i18n-translate');
+module.exports = function (request, cb) {
+  var preloaded = {};
+  var defaultLanguage = 'en_US';
+  request.seneca.act({role: 'cd-dojos', cmd: 'find',
+    query: {urlSlug: request.params['id'] + '/' + request.params['alpha2']}},
+    function (err, dojo) {
+      if (err || !dojo) return cb(); // If metadata fails to load, continue loading the page normally
+      var locale = dojo.alpha2.toLowerCase() + '_' + dojo.alpha2 || defaultLanguage;
+      preloaded.title = translater(locale, {key: 'CoderDojo %1s | Zen',
```

Comment:
Perhaps @rosalanghammer will have some input. I'd be inclined to go for just `'%1s | CoderDojo'` personally. A decent amount of Dojos have "CoderDojo" in their name, so prefixing with CoderDojo would cause weirdness.